x/distribution/types/proposal.go

```diff
@@ -18,7 +20,7 @@ func (csp *CommunityPoolSpendProposal) ProposalType() string { return "Community
 
 // ValidateBasic runs basic stateless validity checks
 func (csp *CommunityPoolSpendProposal) ValidateBasic() error {
```

Comment:
is this still needed if we got rid of the content type?

Reply:
don't think so